src/core/include/openvino/op/rdft.hpp

```diff
@@ -0,0 +1,55 @@
+//*****************************************************************************
```

Comment:
Please update copyright message in all files

Reply:
Fixed.